h2o-r/h2o-package/R/import.R

```diff
@@ -367,5 +367,27 @@ h2o.set_s3_credentials <- function(secretKeyId, secretAccessKey){
   parms$secret_access_key = secretAccessKey
   
   res <- .h2o.__remoteSend("PersistS3", method = "POST", .params = parms, h2oRestApiVersion = 3)
-  print("Credentials successfully set.")
+  print("S3 Credentials successfully set.")
+}
+
+#'
+#' Sets AWS credentials for Hadoop S3A dynamically
+#'
+#' Providing incorrect credentials results in an error during the first related S3A import call.
+#'
+#' @param accessKeyId Amazon S3 Secret Key ID (provided by Amazon, corresponds to fs.s3.awsAccessKeyId)
+#' @param secretAccessKey Amazon S3 Secret Access Key (provided by Amazon, corresponds to fs.s3.awsSecretAccessKey)
+#' 
+#' @export
+h2o.set_s3a_credentials <- function(accessKeyId, secretAccessKey){
```

Comment:
according to tidyverse, the args should probably be snake case, but the S3 function above is camel case, so I wanted to keep those two together.